dropboxdrivefs/core.py

```diff
@@ -148,14 +148,15 @@ def __init__(
         **kwargs
     ):
         """
-        Open a file.
+        Open a file, write mode only
+
         Parameters
         ----------
         fs: instance of DropboxDriveFileSystem
         path : str
             file path to inspect in dropbox
         mode: str
-            Normal file modes.'rb', 'wb' or 'ab'
+            most be "wb"
         block_size: int or None
```

Comment:
What happened with the read mode ?

Reply:
In `_open`, read mode always picks a different class.

Reply:
ah yes of course.